merkledag/merkledag.go

```diff
@@ -334,6 +358,7 @@ type nodePromise struct {
 // the first call to Get will block until the Node is received
 // from its internal channels, subsequent calls will return the
 // cached node.
+// TODO: this is a name clash with node.NodeGetter
```

Comment:
I'm removing this interface in https://github.com/ipfs/go-ipld-format/pull/8 (introducing a single, public `NodePromise` type.